Clients/Xamarin.Interactive.Client/Compilation/Roslyn/RoslynCompilationWorkspace.cs

```diff
@@ -398,6 +399,20 @@ public bool HaveAnyLoadDirectiveFilesChanged (DocumentId documentId)
 
             this.includePeImagesInResolution = includePeImagesInResolution;
 
+            var host = ClientApp.SharedInstance.Host;
+            var hostIsWindows = !host.IsMac;
+
+            // On a Windows host, we can skip patching for Console, .NET Core, and WPF, but
+            // must patch for everything else.
+            if (hostIsWindows) {
```

Comment:
`if (!ClientApp.SharedInstance.Host.IsMac)`

Reply:
Why did I write it this way 🤔...probably had a different structure the first time.